test/new-e2e/tests/agent-runtimes/checks/disk/disk_win_test.go

```diff
@@ -23,7 +23,7 @@ func TestWindowsDiskSuite(t *testing.T) {
 	suite := &windowsDiskCheckSuite{
 		diskCheckSuite{
 			descriptor:            e2eos.WindowsDefault,
-			metricCompareFraction: 0.02,
+			metricCompareFraction: 0.05,
```

Comment:
Not blocking, but do we want to increase for all metrics, or could we just increase for the one flaky metric ?

Reply:
I think this will be enough for now. The solution you propose is very good, but I think it would add complexity to the tests, which I'm not sure we need at this point. In any case, a very good observation.